tests/dummy/app/components/custom-footer.js

```diff
@@ -10,6 +13,14 @@ export default class CustomFooter extends Component {
   @argument rowIndex;
   @argument onFooterEvent;
 
+  @computed('column.valuePath', 'rowValue')
+  get footerValue() {
```

Comment:
why do you need this footerValue for this custom component?

Reply:
It's not needed for tests, but the template renders it and it wasn't there. I'd assumed it was meant to be and added it for future tests, because we'll likely want to verify that it works.

Reply:
This is a duplication of `footerValue` in `ember-table-footer`. I think I put it there in the template but did not actually use it. We can pass `footerValue=footerValue` in `ember-table-footer`.

Reply:
I don't think we should do that just to reduce code. Going to merge this, we need to keep moving.

Reply:
That's good to me. We can clean up this later. I did not mean to bikeshed on this.